aspnetcore/fundamentals/http-requests.md

```diff
@@ -91,14 +91,17 @@ Typed clients:
 
 A typed client accepts an `HttpClient` parameter in its constructor:
 
-[!code-csharp[](http-requests/samples/3.x/HttpClientFactorySample/GitHub/GitHubService.cs?name=snippet1&highlight=5)]
-[!INCLUDE[about the series](~/includes/code-comments-loc.md)]
+[!code-csharp[](http-requests/samples/5.x/HttpClientFactorySample/GitHub/GitHubService.cs?name=snippet1&highlight=5)]
 
 In the preceding code:
 
 * The configuration is moved into the typed client.
 * The `HttpClient` object is exposed as a public property.
 
+<!-- 
+The preceding code can be written as: 
+[!code-csharp[](http-requests/samples/5.x/HttpClientFactorySample/GitHub/GitHubService.cs?name=snippet2)]
```

Comment:
@serpent5  perhaps you could PR this and update the code with your #18716 comment With a `=>` and no `async`/`await`, that can be even less to type, but I didn't want to push it. 😄